addons/rating/models/mail_message.py

```diff
@@ -25,3 +25,13 @@ def _search_rating_value(self, operator, operand):
             ('message_id', '!=', False)
         ])
         return [('id', 'in', ratings.mapped('message_id').ids)]
+
+    def message_format(self, format_reply=True):
+        message_values = super().message_format(format_reply)
+        message_rating_mixin = self.filtered(lambda messgae: issubclass(self.pool[messgae.model], self.pool['rating.mixin']))
+        ratings = self.env['rating.rating'].sudo().search([('message_id', 'in', message_rating_mixin.ids), ('consumed', '=', True)])
+        rating_message_mapping = dict((r.message_id.id, r) for r in ratings)
+        for vals in message_values:
+            if vals['id'] in rating_message_mapping:
+                vals['rating_image_url'] = rating_message_mapping[vals['id']].rating_image_url
```

Comment:
change key to match field name in js so that we can get rid of convert data later